### PR TITLE
erl: add page

### PR DESCRIPTION
**Pull request by mira01**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/erl.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs written in Erlang programming language.
```
Comment (sbrl) [inline pages/common/erl.md]: Perhaps ...`in the Erland programming`... would read better?

File: pages/common/erl.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Run sequential erlang program as a common script and then exit:
```
Comment (sbrl) [inline pages/common/erl.md]: Should `erlang` be capitalised here, or should it be lowercase in the description?

File: pages/common/erl.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`erlc *.erl && erl -noshell '{{mymodule:myfunction(arguments)}}, init:stop().'`
+
+- Connect to a running erlang node:
+
+`erl -remsh {{nodename}}@{{hostname}} -sname {{custom_shortname}} -hidden -setcookie {{cookie_of_remote_node}}`
+
+- Tell the erlang shell to load another modules or other versions:
```
Comment (sbrl) [inline pages/common/erl.md]: Perhaps ....`another module or version:` would read better?

File: pages/common/erl.md (hunk 1 above)
Comment (mira01) [reply pages/common/erl.md]: Ok, thanks

File: pages/common/erl.md (hunk 2 above)
Comment (mira01) [reply pages/common/erl.md]: I think capitalising here is better, going to fix it

File: pages/common/erl.md (hunk 3 above)
Comment (mira01) [reply pages/common/erl.md]: Going to reword to "another modules or version". Is it ok?

File: pages/common/erl.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Run sequential Erlang program as a common script and then exit:
+
+`erlc *.erl && erl -noshell '{{mymodule:myfunction(arguments)}}, init:stop().'`
+
+- Connect to a running Erlang node:
+
+`erl -remsh {{nodename}}@{{hostname}} -sname {{custom_shortname}} -hidden -setcookie {{cookie_of_remote_node}}`
+
+- Tell the Erlang shell to load another modules or version:
```
Comment (sbrl) [inline pages/common/erl.md]: Not sure that `modules` needs to be plural here.

Comment (mira01) [reply pages/common/erl.md]: I believe that yes as there is given a path to the directory where multiple files could be found

Comment (sbrl) [reply pages/common/erl.md]: I see. In that case, we'll need to rework it a bit. How about ....`to load modules from a directory:`?

Comment (mira01) [reply pages/common/erl.md]: Thanks, that sounds good.

File: pages/common/erl.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`erl -pa {{directory_with_beam_files}}`
```
[the remark this answers is not in the document]
Comment (mira01) [reply pages/common/erl.md]: Unfortunately it is not, as in erlang there are three directories the user typically interacts with: top level directory, directory with source code and directory with compiled modules (beam files). These directories could, but do not neccessarily be the same.

File: pages/common/erl.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# erl
+
+> Run and manage programs in Erlang programming language.
+
+- Run sequential Erlang program as a common script and then exit:
+
+`erlc *.erl && erl -noshell '{{mymodule:myfunction(arguments)}}, init:stop().'`
```
[the remark this answers is not in the document]
Comment (mira01) [reply pages/common/erl.md]: Thanks, changing to {{files}}

**Comment by agnivade** [comment]
Unless you have any comments, then yes. :)

**Comment by sbrl** [comment]
Merging due to inactivity.

**Pull request merged**